_data/banking.yml

```diff
@@ -663,8 +663,9 @@ websites:
       url: https://seb.se/
       img: seb.png
       tfa: Yes
+      software: Yes
```

Comment:
Looks like the only software 2FA option is BankID which requires SE citizenship.